### DISCUSSION TITLE
Option to function even when network is lost

### DISCUSSION THREAD

**Discussion by chumleychuff** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#624] [Ideas]
Hi - is it feasible to allow the spa to function without network access? My use case is sometimes my router drops connection to the spa and after some time the spa stops accepting physical commands via the pump interface.

The pump will be up and showing it's LayZSpa module SSID.

The use case is I'm not around, router has dropped the device and my kids just want to use the spa via the physical panel 😁

**Comment by visualapproach** [comment]
Yes, uncheck "Enable WiFi Manager AP"
